src/ado/templates/ad-gh-workflows.yaml

```diff
@@ -49,20 +49,21 @@ jobs:
         run: install.packages("devtools")
         shell: Rscript {0}
 
-      - name: Install adodown
-        run: devtools::install_github("arthur-shaw/adodown")
+      - name: Install adodownr
+        run: devtools::install_github("lsms-worldbank/adodownr")
         shell: Rscript {0}
 
       - name: Install here
         run: install.packages("here")
         shell: Rscript {0}
 
       - name: Build site
-        run: adodown::build_site(pkg_dir = here::here(), site_dir = here::here())
+        run: adodownr::build_site(pkg_dir = here::here(), site_dir = here::here())
         shell: Rscript {0}
 
       - name: Deploy to GitHub pages
-        uses: JamesIves/github-pages-deploy-action@4.1.4
+        if: github.event_name != 'pull_request'
```

Comment:
I had removed this from this line from the template file, but I realize now not from the file used for this repo itself. I think this is no longer needed as we are doing this for all commits to `main`, right?

Reply:
You're right. Removing the `if` condition is something I did on `test_gha` but apparently not here.